clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/converters/DestroyTitusJobAtomicOperationConverter.groovy

```diff
@@ -34,27 +33,25 @@ import org.springframework.stereotype.Component
 @TitusOperation(AtomicOperations.DESTROY_JOB)
 class DestroyTitusJobAtomicOperationConverter extends AbstractAtomicOperationsCredentialsSupport {
 
-  private final TitusClientProvider titusClientProvider
   private final TitusJobProvider titusJobProvider
 
   @Autowired
   DestroyTitusJobAtomicOperationConverter(
-    TitusClientProvider titusClientProvider,
     TitusJobProvider titusJobProvider
   ) {
-    this.titusClientProvider = titusClientProvider
     this.titusJobProvider = titusJobProvider
   }
 
   @Override
   AtomicOperation convertOperation(Map input) {
-    new DestroyTitusJobAtomicOperation(titusClientProvider, convertDescription(input))
+    new DestroyTitusJobAtomicOperation(convertDescription(input))
   }
 
   @Override
   DestroyTitusJobDescription convertDescription(Map input) {
     def converted = objectMapper.convertValue(input, DestroyTitusJobDescription)
     converted.credentials = getCredentialsObject(input.credentials as String)
+    converted.account = converted.credentials.name
```

Comment:
@robzienert Added this as `credentials` was `@JsonIgnore` and for some reason, I saw it getting lost during serialization.

Looking at the code that _shouldn't_ be the case right?

Reply:
Apparently a lie, I just re-tested and all is good.